pages/releases/4.3.html

```diff
@@ -0,0 +1,2313 @@
+			<div class="container">
+				<h3>
+					<a href="#xr">XR</a>
+					<div class="section-robot godot-icon-godot" aria-disabled="true"></div>
+				</h3>
+			</div>
+		</div>
+		<div class="container">
+			<div class="release-cards">
+				<div id="xr-2d-content-renders-less-blurry" class="release-card span-2 media-left inverted">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#xr-2d-content-renders-less-blurry">
+										2D content renders less blurry
+									</a>
+								</h4>
+								<div class="c-blockquote">
+									Finally, someone cleaned the glass
+								</div>
+								<div class="c-content">
+									<p>
+										To correct for lens distortion, visuals in 3D space have to be
+										modified by the XR compositor. This extra processing can cause
+										2D content to become a little blurry and harder-to-read, especially
+										the small text found in user interfaces.
+									</p>
+									<p>
+										By adding your UI in an
+										<code class="highlight"><span class="basetype">OpenXRCompositionLayer</span></code>
+										node, you can force the XR compositor to directly render a viewport,
+										leading to crisper and clearer 2D content.
+									</p>
+								</div>
+								<div class="c-link"
+									data-contributors="dsnopek,BastiaanOlij"
+									data-read-more="https://github.com/godotengine/godot/pull/89880"
+								></div>
+							</div>
+						</div>
+						<div class="release-card-media">
+							<video class="release-card-video lazy" playsinline disablepictureinpicture autoplay loop muted
+								poster="/storage/releases/4.3/images/composition-layers.webp">
+								<source data-src="/storage/releases/4.3/video/composition-layers.webm"
+									type="video/webm" />
+							</video>
+						</div>
+					</div>
+				</div> <!-- #xr-2d-content-renders-less-blurry -->
+
+				<div id="xr-standardized-hand-body-face-tracking" class="release-card">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#xr-standardized-hand-body-face-tracking">
+										Standardized hand, body, and face tracking
+									</a>
+								</h4>
+								<div class="c-content">
+									<p>
+										In this release, we introduced a new standardized system for XR hand tracking,
+										which allows developers to use one kind of node (and code) to create a hand
+										tracking experience that will work in multiple XR eco-systems. This is currently
+										supported by both OpenXR and WebXR.
+									</p>
+									<p>
+										Brand new features are systems for body and face tracking, that are built in the
+										same standartized way, but currently only supported in OpenXR on Meta Quest headsets.
```

Comment:
The order of this sentence sounds very unnatural to me with "brand new features are systems for body and face tracking" - it should be flipped around with "body and face tracking systems are brand new features".

Maybe:

```suggestion
										Brand new systems for body and face tracking were also added, that are built in the
										same standartized way, but currently only supported in OpenXR on Meta Quest headsets.
```